pandas/core/window/ewm.py

```diff
@@ -0,0 +1,404 @@
+from textwrap import dedent
+
+import numpy as np
+
+import pandas._libs.window as libwindow
+from pandas.compat.numpy import function as nv
+from pandas.util._decorators import Appender, Substitution
+
+from pandas.core.dtypes.generic import ABCDataFrame, ABCSeries
+
+from pandas.core.base import DataError
+from pandas.core.window.helpers import _get_center_of_mass
+from pandas.core.window.rolling import (
```

Comment:
could these be moved to `pandas.core.window.helpers` and perhaps renamed `pandas.core.window.common`? Could be follow-up?

Reply:
Good call. Renamed and moved.